web/turkey/survey/static/survey/js/auditors/keypresses_specific.js

```diff
@@ -0,0 +1,24 @@
+var AuditorKeypressesSpecific = {
+    keypresses_specific: [],
+    log_keypress_content: function (e) {
+        this.keypresses_specific.push(String.fromCharCode(e.which));
+    },
+    submit_callable: function () {
+        return {
+            'content':  this.keypresses_specific
```

Comment:
This is going to need some kind of modification vis a vis conversion to Python objects unless you want to write custom code A LOT. Which seems like an unnecessary way to muddy abstractions because we can do what I've seen so far without breaking abstractions.

So I think this one is less straightforward, particularly because I'm not sure how much resolution we want on this data. I see two extremes right now, though I'm not sure either one is the right idea.
One-
Model:
    content = models.TextField() # TextField of all keypresses

and being one model per HIT

Or -
Model:
    keypress = models.IntegerField() # e.which
    timestamp = models.IntegerField() # when user pressed